integration_tests/persistent-workers-test/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@nativelink/persistent-workers-test",
+  "author": "NativeLink Authors",
+  "dependencies": {
+    "@grpc/grpc-js": "^1.9.0",
+    "@grpc/proto-loader": "^0.7.0",
+    "protobufjs": "^7.2.0",
+    "uuid": "^9.0.0"
+  },
+  "description": "Integration tests for NativeLink Remote Persistent Workers",
+  "devDependencies": {
+    "@types/jest": "^29.5.0",
+    "@types/node": "^20.0.0",
+    "@types/uuid": "^10.0.0",
+    "@typescript-eslint/eslint-plugin": "^6.0.0",
+    "@typescript-eslint/parser": "^6.0.0",
+    "eslint": "^8.0.0",
+    "jest": "^29.5.0",
+    "ts-jest": "^29.1.0",
+    "typescript": "^5.0.0"
+  },
+  "keywords": [
+    "nativelink",
+    "persistent-workers",
+    "remote-execution",
+    "bazel"
+  ],
+  "license": "FSL-1.1-Apache-2.0",
+  "main": "dist/index.js",
+  "scripts": {
+    "build": "tsc",
+    "lint": "eslint src/**/*.ts",
+    "test": "jest",
+    "test:integration": "npm run build && node dist/integration.test.js",
```

Comment:
This file doesn't exist. Maybe was meant to be `e2e.test.ts`? But replacing with that doesn't work either. `ts-node src/e2e.test.ts` possibly, after installing `ts-node` but that doesn't seem to fully work.